### ISSUE TITLE
No luck importing animations via FBX

### ISSUE THREAD

**Issue by Lumabrik** [jtydhr88/sd-3dmodel-loader#27]
Is there a particular type of FBX this extension requires to  read the animation data ?
Trying to get animation to play from a DAZ3D exported FBX file, which offers various versions of FBX  from 2006 > 2020, in both binary and ascii flavours.

Thanks.

**Comment by Lumabrik** [comment]
Yes, it looks like there are no  private messages on here? but I can send you a download link, if you can give me an email ?

**Comment by Simorg2002** [comment]
Same issue, but it worked by selecting to download the FBX 'with skin'. Not really sure what daz3d is tho, I downloaded from mixamo.

**Comment by Simorg2002** [comment]
Sure, give me a moment, upload here is slow as hell.
